r/R/array-data.R

```diff
@@ -44,7 +44,7 @@
 ArrayData <- R6Class("ArrayData",
   inherit = ArrowObject,
   active = list(
-    type = function() ArrayData__get_type(self),
+    type =function() ArrayData__get_type(self),
```

Comment:
Just adding a comment here to remind us to remove this once the PR is ready to be approved